http/jetty12/src/main/java/org/apache/felix/http/jetty/internal/JettyConfig.java

```diff
@@ -509,6 +512,10 @@ public int getResponseSizeLimit()
         return getIntProperty(FELIX_JETTY_RESPONSE_SIZE_LIMIT, -1);
     }
 
+    public String getFelixJettyErrorPageCustomHeaders() {
+        return getProperty(FELIX_JETTY_ERROR_PAGE_CUSTOM_HEADERS, null);
```

Comment:
Not using `getStringArray` as splitting on "," is not safe for header values; these might contain "," in the value as well. For example:

`Expires=Thu, 01 Dec 1994 16:00:00 GMT`